packages/webapp/src/utils/__tests__/credits-application.spec.js

```diff
@@ -7,11 +7,17 @@ import applicant from '../../__mocks__/applicant'
 describe('credits-application', () => {
   it('Should process typical application based on test data including LPA code', () => {
     const session = setCreditsApplicationSession()
+
+    session.set(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_APPLICATION_REFERENCE, 'BNGCRD-L4XCQ-AIZMO')
+    session.set(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_PLANNING_AUTHORITY_LIST, 'Hartlepool LPA')
+    session.set(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_DEVELOPMENT_NAME, 'Anything')
+
     const app = creditsApplication(session, applicant)
 
     expect(app.creditsPurchase.creditReference).toEqual('BNGCRD-L4XCQ-AIZMO')
-    expect(app.creditsPurchase.development.localPlanningAuthority.code).toEqual('E60000003')
     expect(app.creditsPurchase.development.localPlanningAuthority.name).toEqual('Hartlepool LPA')
+    expect(app.creditsPurchase.development.localPlanningAuthority.code).toEqual('E60000003')
```

Comment:
Also check the expected value for `app.creditsPurchase.development.planningReference`